pytools/graph.py

```diff
@@ -397,6 +398,50 @@ def compute_induced_subgraph(graph: Mapping[T, Set[T]],
 # }}}
 
 
+# {{{ get_graph_dot_code
+
+def get_graph_dot_code(graph: Mapping[T, Collection[T]]) -> str:
```

Comment:
Maybe support labeling the edges, to support the functionality we discussed Friday.

Reply:
Added in [258d7e8](https://github.com/inducer/pytools/pull/151/commits/258d7e87db0462f15e9198242fbb236113343bc5)